src/datastore/postgres-store.ts

```diff
@@ -5459,18 +5464,22 @@ export class PgDataStore
         WITH
         -- getAddressTxs
         stx_txs AS (
-          SELECT tx_id, ${COUNT_COLUMN}
+          SELECT tx_id
           FROM principal_stx_txs AS s
           WHERE principal = $1 AND ${blockCond}
+        ` +
+          // TODO: this also needs to sort by microblock_sequence DESC, tx_index DESC, but the
+          // columns don't currently exist on the table. this will be a breaking change to fix
```

Comment:
Opened an issue for this TODO https://github.com/hirosystems/stacks-blockchain-api/issues/1050